assets/onboarding/ready/index.jsx

```diff
@@ -0,0 +1,73 @@
+import { Card, H, Link, List, Section } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { MailingListSignupForm } from './mailinglist-signup-form';
+import { formatString } from '../helpers/format-string.js';
+
+/**
+ * Ready step for Setup Wizard.
+ */
+export const Ready = () => (
+	<>
+		<div className="sensei-onboarding__title">
+			<H>
+				{ __(
+					`You're ready to start creating online courses!`,
+					'sensei-lms'
+				) }
+			</H>
+		</div>
+		<Card className="sensei-onboarding__card">
+			<Section className="sensei-onboarding__mailinglist-signup">
+				<H>{ __( `Join our mailing list`, 'sensei-lms' ) }</H>
+				<p>
+					{ __(
+						`We're here for you — Get tips, product updates, and inspiration straight to your mailbox.`,
```

Comment:
Should be "get" vs. "Get" as this is the continuation of the same sentence.

Reply:
Updated in 562d2bdc.